poses:
  home:
    torso_1_joint:     0.0
    torso_2_joint:     0.0
    head_1_joint:      0.0
    head_2_joint:      0.0
    arm_right_1_joint: 0.0
    arm_right_2_joint: 0.0
    arm_right_3_joint: 0.0
    arm_right_4_joint: 0.0
    arm_right_5_joint: 0.0
    arm_right_6_joint: 0.0
    arm_right_7_joint: 0.0
    arm_left_1_joint:  0.0
    arm_left_2_joint:  0.0
    arm_left_3_joint:  0.0
    arm_left_4_joint:  0.0
    arm_left_5_joint:  0.0
    arm_left_6_joint:  0.0
    arm_left_7_joint:  0.0

  init:
    torso_1_joint:      0.0
    torso_2_joint:      0.0
    head_1_joint:       0.0
    head_2_joint:       0.0
    arm_right_1_joint: -0.4
    arm_right_2_joint:  0.1
    arm_right_3_joint: -0.1
    arm_right_4_joint:  0.6109
    arm_right_5_joint:  0.0
    arm_right_6_joint:  0.0
    arm_right_7_joint:  0.0
    arm_left_1_joint:  -0.4
    arm_left_2_joint:   0.1
    arm_left_3_joint:  -0.1
    arm_left_4_joint:   0.6109
    arm_left_5_joint:   0.0
    arm_left_6_joint:   0.0
    arm_left_7_joint:   0.0

  surrender:
    torso_1_joint:     0.0
    torso_2_joint:     0.0
    head_1_joint:      0.0
    head_2_joint:      0.0
    arm_right_1_joint: 0.0
    arm_right_2_joint: 1.5708
    arm_right_3_joint: 1.5708
    arm_right_4_joint: 1.5708
    arm_right_5_joint: 0.0
    arm_right_6_joint: 0.0
    arm_right_7_joint: 0.0
    arm_left_1_joint:  0.0
    arm_left_2_joint:  1.5708
    arm_left_3_joint:  1.5708
    arm_left_4_joint:  1.5708
    arm_left_5_joint:  0.0
    arm_left_6_joint:  0.0
    arm_left_7_joint:  0.0

  hands_up:
    torso_1_joint:      0.0
    torso_2_joint:      0.0
    head_1_joint:       0.0
    head_2_joint:       0.34
    arm_right_1_joint:  1.2
    arm_right_2_joint:  0.0
    arm_right_3_joint: -1.6
    arm_right_4_joint:  1.4
    arm_right_5_joint: -0.5
    arm_right_6_joint:  0.0
    arm_right_7_joint:  0.0
    arm_left_1_joint:   0.9
    arm_left_2_joint:   0.2
    arm_left_3_joint:  -1.6
    arm_left_4_joint:   1.3
    arm_left_5_joint:   -0.5

  raise_left_hand:
    torso_1_joint:      0.0
    torso_2_joint:      0.0
    head_1_joint:       0.0
    head_2_joint:       0.34
    arm_right_1_joint:  1.2
    arm_right_2_joint:  0.0
    arm_right_3_joint: -1.6
    arm_right_4_joint:  1.4
    arm_right_5_joint: -0.5
    arm_left_1_joint:   1.7
    arm_left_2_joint:  -0.1
    arm_left_3_joint:  -1.7
    arm_left_4_joint:   1.5708
    arm_left_5_joint:   0.0
    arm_left_6_joint:   0.0
    arm_left_7_joint:   0.0

